Service/EnvironmentService.php

```diff
@@ -204,4 +316,11 @@ public function getAllInMyCircle()
             return count(array_intersect($user->getCircles(), $environment->getCircles())) >= 1;
         });
     }
+
+    public function clearCache()
```

Comment:
Not needed (to be removed) when using #281 maybe we should add an @deprecated to this function immediatly and reference 281 in it?